src/components/Match/matchColumns.jsx

```diff
@@ -374,14 +375,48 @@ export default (strings) => {
           </div>,
       }
       : [])
+      .concat(
+        {
+          paddingLeft: 5,
+          paddingRight: 0,
+          width: 32,
+          displayFn: row => {
+            const shardTooltip = <ItemTooltip item={items.aghanims_shard}/>;
+            const aghTooltip = <ItemTooltip item={items.ultimate_scepter}/>
+            return (
+              <StyledAghanimsBuffs>
+                <ReactTooltip id="agh" effect="solid" place="left">
+                  {aghTooltip}
+                </ReactTooltip>
+                <ReactTooltip id="shard" effect="solid" place="left">
+                  {shardTooltip}
+                </ReactTooltip>
+                <img
+                  src={`/assets/images/dota2/agh_${row.permanent_buffs && row.permanent_buffs.some(b => b.permanent_buff === 2) ? '1' : '0'}.png`}
+                  alt=""
+                  data-tip={aghTooltip}
+                  data-for="agh"
+                />
+                <img
+                  src={`/assets/images/dota2/shard_${row.permanent_buffs && row.permanent_buffs.some(b => b.permanent_buff === 12) ? '1' : '0'}.png`}
+                  alt=""
+                  data-tip={shardTooltip}
+                  data-for="shard"
+                />        
+              </StyledAghanimsBuffs>  
+            )
+          }
+        },
+      )
       .concat(match.players.map(player => player.permanent_buffs && player.permanent_buffs.length).reduce(sum, 0) > 0
         ? {
           displayName: strings.th_permanent_buffs,
           tooltip: strings.tooltip_permanent_buffs,
           field: 'permanent_buffs',
           width: 135,
           displayFn: row =>
-            (row.permanent_buffs && row.permanent_buffs.length > 0 ? row.permanent_buffs.map(buff => inflictorWithValue(buffs[buff.permanent_buff], buff.stack_count, 'buff')) : '-'),
+            (row.permanent_buffs && row.permanent_buffs.length > 0 ? row.permanent_buffs
+            .filter(b => b.permanent_buff !== 2 && b.permanent_buff !== 12).map(buff => inflictorWithValue(buffs[buff.permanent_buff], buff.stack_count, 'buff')) : '-'),
```

Comment:
probably want to put 2 and 12 into named constants for readability

Reply:
done